packages/protobuf/src/reflect/reflect.ts

```diff
@@ -224,64 +278,88 @@ function assertOwn(owner: Message, member: DescField | DescOneof) {
 export function reflectList<V>(
   field: DescField & { fieldKind: "list" },
   unsafeInput?: unknown[],
+  check = true,
```

Comment:
This is new, and not final yet, see existing comment on line 51. We're not using it yet.